js/terms.json

```diff
@@ -77,14 +77,14 @@
    },
    {
     "glossary-term": "Independent expenditure",
-    "glossary-definition": "An independent expenditure is an expenditure for a communication \"expressly advocating the election or defeat of a clearly identified candidate that is not made in cooperation, consultation, or concert with, or at the request or suggestion of, a candidate, a candidate’s authorized committee, or their agents, or a political party or its agents.\" <a href='http://www.ecfr.gov/cgi-bin/text-idx?SID=a8f74e22426c2a67f75a4884e2b2348f&mc=true&node=se11.1.100_116&rgn=div8'>11 CFR 100.16(a)</a>."
+    "glossary-definition": "An expenditure for a communication that expressly advocates the election or defeat of a clearly identified candidate and that is not made in cooperation, consultation or concert with, or at the request or suggestion of, any candidate, or his or her authorized committees or agents, or a political party committee or its agents. 11 CFR 100.16."
    },
    {
-    "glossary-term": "Electioneering",
-    "glossary-definition": "An electioneering communication is any broadcast, cable or satellite communication that fulfills <em>each</em> of the following conditions:<ul class='list--bulleted'><li>The communication refers to a clearly identified candidate for federal office;</li><li>The communication is publicly distributed shortly before an election for the office that candidate is seeking; and</li><li>The communication is targeted to the relevant electorate (U.S. House and Senate candidates only).</li></ul>"
+    "glossary-term": "Electioneering communication",
+    "glossary-definition": "Any broadcast, cable or satellite communication that: <ul><li>Refers to a clearly identified candidate for federal office;</li><li>Is publicly distributed within certain time periods before an election; and</li><li>Is targeted to the relevant electorate.</li></ul>11 CFR 100.29."
    },
    {
     "glossary-term": "Communication cost",
-    "glossary-definition": "52 U.S.C. 30118 allows \"communications by a corporation to its stockholders and executive or administrative personnel and their families or by a labor organization to its members and their families on any subject,\" including the express advocacy of the election or defeat of any Federal candidate. The costs of such communications must be reported to the Federal Election Commission under certain circumstances."
+    "glossary-definition": "52 U.S.C. 30118 allows \"communications by a corporation to its stockholders and executive or administrative personnel and their families or by a labor organization to its members and their families on any subject,\" including the express advocacy of the election or defeat of any federal candidate. The costs of such communications must be reported to the federal Election Commission under certain circumstances."
```

Comment:
Isn't "Federal Election Commission" a proper noun? We aren't talking about an Election Commission that happens to be federal.